tests/services/test_job_execution.py

```diff
@@ -279,4 +368,59 @@ async def test_access_control_job_execution_with_dls_feature_flag_disabled(
 
     await create_and_run_service()
 
-    concurrent_tasks_mock.put.assert_awaited()
+    content_syncs_tasks_mock.put.assert_not_awaited()
+    access_control_syncs_tasks_mock.put.assert_not_awaited()
+
+
+@pytest.mark.asyncio
+async def test_job_execution_with_unknown_job_type(
```

Comment:
Can we also assert the relevant error is logged?

Reply:
We've decided to stay away from using assertions on log messages for several reasons (more fragile tests, focus should be on the functionality, in this case "not executing a job" etc.). So, I'm hesitant to reintroduce it here.

WDYT?